src/reducers/teamReducer.js

```diff
@@ -0,0 +1,27 @@
+import { LOAD_TEAMS_SUCCESS, TOGGLE_TEAM } from '../actions/actionTypes';
+import initialState from './initialState';
+
+const team = (state = {}, action) => {
+  switch(action.type) {
+    case TOGGLE_TEAM:
+      debugger
+      const regex = new RegExp(`(${action.team_name})`, "i");
```

Comment:
I am not a fan of logic in reducer. It should be in the action.
Also why you need the regex?